www/main/viewer/Slide/Slide.jsx

```diff
@@ -191,7 +208,9 @@ const Slide = ({ verseObj, nextLineObj, isMiscSlide, bgColor, updateVerseRef })
       </CSSTransition>
     </div>
   );
-};
+});
+
+Slide.displayName = 'Slide';
```

Comment:
@amitojsingh veerji, where do we use this displayName?

Reply:
@amitojsingh ping

Reply:
DisplayName allows you to name your component. This name is used by React in debugging messages.
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/display-name.md